site/layouts/partials/footer.html

```diff
@@ -4,17 +4,17 @@
 
 <footer class="footer navbar bd-footer" data-bs-theme="dark">
   <h2 class="visually-hidden">OUDS Web sitemap & information</h2>
-  <div class="container-xxl footer-nav">
+  <div class="container-fluid max-width-specific-tools footer-nav">
     <nav class="accordion" id="sitemapAccordion" aria-label="OUDS Web sitemap footer">
       <div class="row">
         <div class="footer-column col-md-6 col-lg-2">
           <h3 class="accordion-header footer-heading" id="headingLinks">
-            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button" data-bs-toggle="collapse" data-bs-target="#collapseLinks" aria-expanded="true" aria-controls="collapseLinks">
+            <button class="accordion-button collapsed container-fluid max-width-specific-tools px-1 d-md-none" type="button" data-bs-toggle="collapse" data-bs-target="#collapseLinks" aria-expanded="true" aria-controls="collapseLinks">
```

Comment:
Are we sure this is the right modification to do here?

![Screenshot 2024-10-16 at 12 04 02](https://github.com/user-attachments/assets/954d280a-29e3-4a49-be88-9cb446b92a52)